docs/src/pages/guides/auth-protected/auth-protected.react.mdx

```diff
@@ -0,0 +1,414 @@
+import { Image } from '@aws-amplify/ui-react';
+
+The [Amplify UI Authenticator](/components/authenticator) component allows you to quickly scaffold out an end-to-end authentication flow, in
+just a few lines of code.
+
+Typically, this can be accomplished by wrapping the entire application in the [withAuthenticator](components/authenticator?platform=react#quick-start) higher-order component.
+This forces the entire application behind a login page.
+
+Let's imagine you needed a web application where the login page was in it's own route, and that you needed a handful of selected routes
+to only be accessed by logged in users.
```

Comment:
Looks like there's a newline here, this should be removed and let markdown handle line breaks.

Reply:
I was creating my own line breaks so it wasn't one super long line. But maybe my editor will do it automatically?

Reply:
7c6ee573f1e15b8407892dfd8552ea416e8cdf63

Reply:
VS code has "Toggle word wrap" option that I like to use for markdown files 🙏

Reply:
Nice! That works :)

Reply:
oh that is so nice, how have I gone this long without knowing about that